2016-08-21-Crash-Reporter.md

```diff
@@ -0,0 +1,35 @@
+### The Tessel Crash Reporter service
+
+The Tessel 2 [crash reporter](http://crash-reporter.tessel.io) is a web service which automatically collects anonymized crash reports when a crash is detected in the Tessel 2's command line interface (CLI). Given that the CLI is one of the primary ways of interacting with your Tessel it is important that the CLI be extremely stable and a joy to use.
+
+This project started during the early days of the CLI. We wanted to make sure that the team was proactively looking at, and fixing issues that users could potentially run into. We wanted the onboarding experience was as smooth as possible for new users.
+
+#### How does it work
+
+Every time you use the Tessel 2 CLI, a central controller dispatches your command to an appropriate task handler. The crash reporter automatically checks for uncaught `Errors` and unresolved `Promises` in the node process that's executing the task. 
```

Comment:
Touch on the privacy issues we worked around (users can decide to turn it on/off, their choice)?